### PR TITLE
docs(fr): Fix typo

### PR DESCRIPTION
**Pull request by vaga**

### CODE REVIEW

**Comment by kiaking** [comment]
@posva Could you please review this one 🙏

File: docs/fr/guide/mutations.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Puisqu'un état de store de Vuex est rendu réactif par Vue, lorsque nous mutons
 
   - Utiliser `Vue.set(obj, 'newProp', 123)`, ou
 
-  - Remplacer cet objet par un nouvel objet. Par exemple, en utilisant [opérateur de décomposition](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Op%C3%A9rateur_de_d%C3%A9composition) (stage-2), il est possible d'écrire :
+  - Remplacer cet objet par un nouvel objet. Par exemple, en utilisant un [opérateur de décomposition](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Op%C3%A9rateur_de_d%C3%A9composition) (stage-2), il est possible d'écrire :
```
Comment (posva) [inline docs/fr/guide/mutations.md:91]:
```suggestion
  - Remplacer cet objet par un nouvel objet. Par exemple, en utilisant l'[opérateur de décomposition](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Syntaxe_d%C3%A9composition#Utiliser_la_d%C3%A9composition_avec_les_litt%C3%A9raux_objet) (stage-2), il est possible d'écrire :
```

**Comment by kiaking** [comment]
@vaga Thanks! Could you please rebase this branch the latest `dev` to resolve the conflicts as well 🙏

**Comment by kiaking** [comment]
@vaga Thank you so much! 🙌

**Pull request merged**